README.md

```diff
@@ -21,6 +21,16 @@ The solution will be created in current folder with name of this folder. For set
 dotnet new logofx-wpf -n <new-solution-name>
 ```
 
+## Add model entitie to existing project
+
+Inside solution folder run
+
+```
+dotnet new logofx-model -sn <solution name> -n <model name>
+```
+
+This command adds `<model name>` model to `<solution name>.Model` project; `I<model name>` constract to `<solution name>.Model.Contracts` project; `<model name>Dto` dto to `<solution name>.Data.Contracts.Dto` project.
```

Comment:
Add the instructions on manual completion of the process:
Module and MappingProfile updates

Reply:
Instruction added only for MappingProfile. Module is already good after creating a solution.

Reply:
Not exactly. After generating a new `<model name>Mapper` user needs to register it into the container (this is added to the `Module`) and then it will be available for injection:
```
dependencyRegistrator
                ....
                .AddSingleton<<model name>Mapper>();
```

Reply:
Another q: Why do the folders for model template start with `LogoFX.Templates.WPF` and not with `LogoFX.Templates`?